modules/registry-configuring-storage-azure-user-infra.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring-registry-storage-azure.adoc
+
+[id="registry-configuring-storage-azure-user-infra_{context}"]
+= Configuring registry storage for Azure
+
+During installation, your cloud credentials are sufficient to create Azure Blob
+Storage and the Registry Operator will automatically configure storage. Azure
+registry storage uses the struct, `ImageRegistryConfigStorageAzure`.
+
+.Prerequisites
+
+* A cluster on Azure with user-provisioned infrastructure.
+* To configure registry storage for Azure, you need to provide Registry Operator
+cloud credentials.
+* For Azure storage the Secret is expected to contain one key:
+** `REGISTRY_STORAGE_AZURE_ACCOUNTKEY`
+
+.Procedure
+
+ * Create an Azure storage container and provide the `accountName` and
```

Comment:
Do you have a link in the Azure docs that describes this process? If so, I'd add it.